application.js

```diff
@@ -21,25 +21,7 @@ class Application extends App {
     this.env = process.env.NODE_ENV // eslint-disable-line no-process-env
       || options.env
       || 'development';
-
-    this.after('close', onClose);
```

Comment:
Is there anything we need to do to optimistically shutdown the dynamodb connection?

Reply:
na its just http